nixos/modules/virtualisation/libvirtd.nix

```diff
@@ -23,6 +23,9 @@ let
     ''}
     ${cfg.qemuVerbatimConfig}
   '';
+  dir = "/var/lib/${dirName}";
+  dirName = "libvirt";
```

Comment:
Any reason to introduce this level of indirection?
Is it a preparation for running multiple ```libvirt``` services simulteniously, each with own ```dirName``` and ```dir``` ?

Reply:
Not a good one, no. Fixed.

Any other concerns?

Reply:
I meant not only ```dir```, but also ```dirName = "libvirt";``` and questionable ```s/libvirt/${dirName}/g``` across the whole file.

Reply:
I'm sure I replied to this earlier. but clearly not. 

I'm always having a hard time guessing when it's ```libvirt``` and when it's ```libvirtd```, so I thought to simply make things easier for the next person by moving into a variable.